### PR TITLE
Proposing some other non-filtering public providers

### PR DESCRIPTION
**Pull request by Shtong**

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Shtong Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/public_dns/providers.yml [hunk 1]
```diff
@@ -41,3 +41,23 @@ DNS.Watch:
   ip6:
     - 2001:1608:10:25::1c04:b12f
     - 2001:1608:10:25::9249:d69b
+FreeDNS:
+  info_url: http://freedns.zone
+  ipv4:
+    - 37.235.1.174
+    - 37.235.1.177
+  ipv6: []
```
Comment (moollaza) [inline share/goodie/public_dns/providers.yml]: I beleive this is causing the test to fail. Please run the t/public_dns.t file locally (or checkout https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/86254558#L4547) to see the failure.

**Comment by moollaza** [comment]
@Shtong once the test pass this LGTM :+1: 

@iambibhas @mintsoft you guys have worked on this before, any thoughts?

**Comment by Shtong** [comment]
Duh, my bad. I fixed my typos and will now go back to reading school :)

**Comment by moollaza** [comment]
@Shtong thanks! this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**